frontend/src/chain/utils/getBlockExplorerUrl.ts

```diff
@@ -0,0 +1,51 @@
+import { match } from '../../lib/utils/match';
+import { Chain } from '../../model/chain';
+
+type ChainEntity = 'address';
+
+type GetBlockExplorerUrlInput = {
+  chainId: Chain;
+  entity: ChainEntity;
+  value: string;
+};
+
+export const getBlockExplorerUrl = ({
+  chainId,
```

Comment:
we have a method call `getExplorerByAddressURL` in  Endpoint.ts already , shall we use that one ?
or you can update that function

Reply:
Good catch. I removed the function from the `Endpoint` class, first this fie is too big already, second, concept of `Endpoint` is too broad.